docs/topics/api-clients.md

```diff
@@ -428,6 +428,9 @@ the user to login, and then instantiate a client using session authentication:
 The authentication scheme will handle including a CSRF header in any outgoing
 requests for unsafe HTTP methods.
 
+** Note: ** This mechanism does not work when used in conjunction with
+`CSRF_USE_SESSIONS = True` in your Django settings.
+
```

Comment:
I think drop this block. I'd rather address it properly if it comes up (which it may not given 3.9+ moves towards using APIStar as the client library.)